src/NSubstitute/NSubstitute.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
```

Comment:
**question:** does dropping `net462` gain us a lot? I remember being stuck on older .NET versions at a previous job and it was really frustrating to not be able to use certain tools, so it would be great to have as broad support as we can without it being prohibitively difficult to maintain.

Reply:
.NET Standard have included support for net462. We run tests for net 4.6.2 and this is still supproted

Reply:
I'd prefer to run on as many .NET versions as we can (even deprecated) unless/until it becomes a lot of work to maintain that (even if we don't explicitly test, we just don't drop target). Agree they can use an older version, but if it isn't adding extra work to keep support then I'd like to.